### DISCUSSION TITLE
Domanda sull'uso dei moduli di richiesta

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#410] [Richieste di aiuto] [answered]
Buongiorno, volevo sapere se fosse possibile modificare i termini massimi per l'invio delle richieste, che ora è alle ore 16:00 del giorno precedente.

**Comment by trinko** [comment, marked as answer]
Si, devi modificare il parametro di configurazione **scadenza_invio_richiesta**

**Reply by Lucaccc** [reply]
> Si, devi modificare il parametro di configurazione **scadenza_invio_richiesta**

Grazie mille.
